src/pages/api/webhooks/saleor/payment-gateway-initialize-session.ts

```diff
@@ -21,7 +22,7 @@ export const paymentGatewayInitializeSessionSyncWebhook =
     apl: saleorApp.apl,
     event: "PAYMENT_GATEWAY_INITIALIZE_SESSION",
     query: UntypedPaymentGatewayInitializeSessionDocument,
-    webhookPath: "/api/webhooks/saleor/payment-gateway-initialize-session",
+    webhookPath: `${env.NEXT_PUBLIC_BASE_PATH}/api/webhooks/saleor/payment-gateway-initialize-session`,
```

Comment:
```suggestion
    webhookPath: `/api/webhooks/saleor/payment-gateway-initialize-session`,
```

Urls in the webhook definitions should not be changed. The full URL is constructed in the createManifestHandler (https://github.com/saleor/saleor-app-payment-klarna/pull/65/files#diff-8b106ce88b28921d34154dd6adf1dca085721efccd424f05bc739b52c5bf72baR33). In manifest file comment provided a better solution.